test/suspend.js

```diff
@@ -0,0 +1,68 @@
+import {serial as test} from 'ava';
+import sinon from 'sinon';
+
+const mockery = require('mockery');
+
+const logUpdateApi = {
+	main(text) {
+		console.log(text);
+	}
+};
+logUpdateApi.main.clear = function () {
```

Comment:
You should create a noop function (e.g. `const noop = () => {};`) and use it here and below.

Reply:
I imported the `log-update`-module directly. This section now should be refactored.